playbooks/dropbear_initramfs_update.yaml

```diff
@@ -0,0 +1,30 @@
+# vim:ts=2:sw=2:et:ai:sts=2
+
+# Playbook: Update dropbear initramfs authorized key.
+---
+- name: Update dropbear initramfs authorized keys
+  hosts: libvirt_hosts
+  become: true
+  vars:
+    authorized_keys_file: '/etc/dropbear/initramfs/authorized_keys'
```

Comment:
This default would be better placed in the `playbooks/group_vars/all/13-dropbear_initrfs.yaml` file, so it can be overridden by local configuration